.github/renovate.json

```diff
@@ -1,7 +1,10 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "schedule": "after 7am and before 11am every weekday",
-  "extends": ["config:base"],
+  "extends": [
+    "local>hmcts/.github:renovate-config",
+    "local>hmcts/.github//renovate/automerge-all"
+  ],
```

Comment:
I think this is ok for this project.  For others I think we may have to tweak automerge-all